contracts/common/Singleton.sol

```diff
@@ -1,11 +1,13 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity >=0.7.0 <0.9.0;
 
-/// @title Singleton - Base for singleton contracts (should always be first super contract)
-///         This contract is tightly coupled to our proxy contract (see `proxies/SafeProxy.sol`)
-/// @author Richard Meissner - <richard@gnosis.io>
+/**
+ * @title Singleton - Base for singleton contracts (should always be first super contract)
+ *        This contract is tightly coupled to our proxy contract (see `proxies/SafeProxy.sol`)
+ * @author Richard Meissner - @rmeissner
+ */
 contract Singleton {
-    // singleton always needs to be first declared variable, to ensure that it is at the same location as in the Proxy contract.
-    // It should also always be ensured that the address is stored alone (uses a full word)
+    // The singleton always has to be the first declared variable to ensure the same location as in the Proxy contract.
```

Comment:
I think `The singleton` sounds confusing here, I would leave it as `singleton`

Reply:
It's referring to one particular variable, so I think using `the` is correct from the language standpoint. We can ask around some Englishmen

Reply:
The other comments aren't using definite articles ("the") so I would suggest leaving this as it was.